Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,10 @@
 
 /* Begin PBXBuildFile section */
 		023057F22ACFAD79006C5A73 /* EditEventDetailView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 023057F12ACFAD79006C5A73 /* EditEventDetailView.swift */; };
+		8014A4CD2AD755C8005B51F6 /* Basic-Car-Maintenance.xcconfig in Resources */ = {isa = PBXBuildFile; fileRef = FF098EFA2AB3424E003EC0FE /* Basic-Car-Maintenance.xcconfig */; };
```

Comment:
It looks like you committed the `Basic-Car-Maintenance.xcconfig` file when you shouldn't have. See the [README.md](https://github.com/mikaelacaron/Basic-Car-Maintenance#getting-started) for instructions on how to copy this file properly

Reply:
I must have misunderstood something. I did run `cp Basic-Car-Maintenance.xcconfig.template Basic-Car-Maintenance.xcconfig` like the README says. 

The file itself is not in my commit history. I guess you mean that I shouldn't have added `Basic-Car-Maintenance.xcconfig` to the .xcodeproj?

Is that the reason why you had `#include "Basic-Car-Maintenance.xcconfig"` in the `Project.xcconfig`. If so, I think something must be set up wrong on my machine because the `#include` wasn't finding the file, and the project wasn't building until I added the `Basic-Car-Maintenance.xcconfig` to the project. 

However it does appear to be building now (even after removing `Basic-Car-Maintenance.xcconfig`) so I guess Xcode was building based off of a stale index.

Reply:
You should run `cp Basic-Car-Maintenance.xcconfig.template Basic-Car-Maintenance.xcconfig` while being in the root directory (where the xcodeproj is), cause  `Basic-Car-Maintenance.xcconfig` is in the `.gitignore`

The include is to override values from the main config and also use them in the Project.xcconfig

And you run that command, before opening Xcode!

Reply:
Ok it sounds like the issue was that I opened Xcode before running the command. Lesson learned.

Reply:
I'll add this to the README for more clarification 👍